### PR TITLE
Login debug

### PR DESCRIPTION
**Pull request by czye17**
Warning: Updated CASClient.py on server side because running in Dev requires changing of the certificate. We should probably discard any changes made in the file but I have pushed it here just in case we need the fix in the future.

### CODE REVIEW

File: clikr-server/requirements.txt [hunk 1]
```diff
@@ -1,33 +1,32 @@
-alembic==1.4.2
-bcrypt==3.2.0
-cffi==1.14.2
-click==7.1.2
+alembic==1.0.5
```
Comment (cakirmehmete) [inline clikr-server/requirements.txt:1]: can you remove these changes to the requirement.txt? It seems like its changing the formatting so its unnecessary.

File: clikr-server/app/models/StudentModel.py [hunk 2]
```diff
@@ -2,8 +2,9 @@
 from marshmallow import fields, Schema
 import datetime
 from .. import db
-from ..shared.Util import CustomStringField, CustomDateTimeField
+from ..shared.Util import CustomStringField, CustomDateTimeField, generate_salt
```
Comment (cakirmehmete) [inline clikr-server/app/models/StudentModel.py:5]: why is the salt being added back in?

File: clikr-server/app/shared/CASClient.py [hunk 3]
```diff
@@ -34,7 +35,11 @@ def validate(self, ticket):
         val_url = self.cas_url + 'validate' + \
             '?service=' + urllib.parse.quote(self.strip_ticket(request)) + \
             '&ticket=' + urllib.parse.quote(ticket)
-        r = urllib.request.urlopen(val_url).readlines() # returns 2 lines # TODO: only for Princeton CAS or always?
+
+        myssl = ssl.create_default_context()
```
Comment (cakirmehmete) [inline clikr-server/app/shared/CASClient.py:39]: is this still needed after rebasing to master? I fixed my local version and it didn't require this.

Comment (czye17) [reply clikr-server/app/shared/CASClient.py:39]: yep don't need these

**Pull request merged**